guide/16-geoenrichment-guides/part4_enrich_collections_and_sedf.ipynb

```diff
@@ -0,0 +1,2925 @@
+{
```

Comment:
<p>Mohit, would be even better if you can please explain what these columns mean?</p><p>esp. what does <code>vintage</code>  mean?</p><p>Also, is "2025 Female Age 75" going to be identical to "2020 Female Age 70"? I am confused here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>